### PR TITLE
update: OS snaptshot data migration feedback

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes
Addressed feedback on snapshot data migration content based on testing 

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

**Review by hoang-vo — APPROVED** [review]

File: docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md [hunk 1]
```diff
@@ -400,7 +400,24 @@ The response shows the snapshot status and details if a snapshot is in progress:
...
+
+Next, reapply any necessary configurations:
+
+- **Reapply ISM policies**: Reapply any Index State Management (ISM) policies to the
+  restored indices. For more information, see
+  [Reapply ISM policies after snapshot restore](/docs/products/opensearch/howto/migrate-ism-policies).
+
+- **Update security configurations**: Review and reconfigure any security settings,
+  including OpenDistro security configurations. For more details, see
+  [Migration Opendistro security configuration](/docs/products/opensearch/howto/migrate-opendistro-security-config-aiven).
```
Comment (staceysalamon-aiven) [inline docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md:418]:
```suggestion
  [Migrate OpenDistro security configuration](/docs/products/opensearch/howto/migrate-opendistro-security-config-aiven).
```

**Review by staceysalamon-aiven — APPROVED** [review]
Looks good! 🚀  Just fixed a small typo!

File: docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md (hunk 1 above)
Comment (wojcik-dorota) [inline docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md]: Nitpicking to remove some potentially redundant word:

```suggestion
Reapply necessary configurations:

- **Reapply ISM policies**: Reapply Index State Management (ISM) policies to the
  restored indices. For more information, see
  [Reapply ISM policies after snapshot restore](/docs/products/opensearch/howto/migrate-ism-policies).

- **Update security configurations**: Review and reconfigure security settings,
  including OpenDistro security configurations. For more details, see
  [Migration Opendistro security configuration](/docs/products/opensearch/howto/migrate-opendistro-security-config-aiven).
```

**Review by wojcik-dorota — APPROVED** [review]
A few super tiny nitpicks :-) LGTM!

File: docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md]: Thanks! Addressed them!

**Pull request merged**